manifest.json

```diff
@@ -22,7 +22,9 @@
         { "path": "./environments/personalization.md" },
         { "path": "./environments/preferences.md" },
         { "path": "./environments/pwa.md" },
-        { "path": "./environments/ssh.md" }
+        { "path": "./environments/ssh.md" },
+        { "path": "./environments/workspaces-as-code.md" },
+        { "path": "./environments/wac-syntax.md" }
```

Comment:
workspace as code should be its own section in the manifest. This is how workspace providers did things:

https://github.com/cdr/docs/blob/2d69f49daca3ee989136269eca4888704e8b22b0/manifest.json#L102-L105

Reply:
I'm taking a look now, but here's how I have it (matches WP):

<img width="263" alt="Screen Shot 2021-03-30 at 10 04 03 AM" src="https://user-images.githubusercontent.com/16521651/113011340-50d47480-913f-11eb-85bb-cdbc3f44a4c2.png">